### ISSUE TITLE
Screen reader announcement on revealing answers doesn't make sense

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#830]
From the [BCCampus accessibility review](https://bccampus.ca/wp-content/uploads/2020/09/MathTest-Accessibility-Review-AC.docx).

When you reveal the answers to a question, apparently the screen reader says "your answer is incorrect", but doesn't read out the correct answer.

I think it should instead say something like "answers have been revealed, check through the question", or it could even read out each answer in turn? Would need to test to find out what makes sense.
